packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -195,7 +196,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
                 onRenderPlaceHolder(this.props, this._onRenderPlaceHolder)
             }
           </span>
-          <Icon className={ css('ms-Dropdown-caretDown', styles.caretDown) } iconName='ChevronDown' />
+          { onRenderChevronDownIcon(this.props, this._onRenderChevronDownIcon) }
```

Comment:
My bad here. I think I mentioned ChevronDown, but let's go with onRenderCaretDown so that we match the className associated with that element.

Reply:
No worries, that makes sense.  I've updated the name to `onRenderCaretDown` as suggested.